client/bundles.py

```diff
@@ -8,10 +8,11 @@
 
 
 class Bundles(dict, Enum):
-    CLI_TOOLS = {'name': 'cli-tools', 'title': 'CLI tools'}
-    DRIVERS = {'name': 'other-drivers', 'title': 'Other drivers'}
-    DRIVER_C = {'name': 'c-driver', 'title': 'C driver'}
-    DRIVER_CPP = {'name': 'cpp-driver', 'title':'C++ driver'}
+    SQL_TOOLS = {"name": "sql-tools", "title": "SQL tools"}
+    CLI_TOOLS = {"name": "cli-tools", "title": "CLI tools"}
+    DRIVERS = {"name": "other-drivers", "title": "Other drivers"}
+    DRIVER_C = {"name": "c-driver", "title": "C driver"}
+    DRIVER_CPP = {"name": "cpp-driver", "title": "C++ driver"}
 
     def __str__(self):
-        return self['name']
+        return self["name"]
```

Comment:
Why this change?

Reply:
My editor likes double quotes and change it on file save. I put it back on my lastest push.